File: src/com/jepl/lang/libraries/math/math.java
package com.jepl.lang.libraries.math;

import com.jepl.lang.Values;
import com.jepl.lang.libraries.Function;
import com.jepl.lang.libraries.Library;

import java.util.HashMap;
import java.util.List;

public class math implements Library {
    @Override
    public HashMap<String, Function> invoke() {
        HashMap<String, Function> out = new HashMap<>();
        out.put("math_random", new Function() {
            @Override
            public void invoke(List<String> args) {
                int range = Integer.parseInt(args.get(1));
                int min = Integer.parseInt(args.get(0));
                String var = args.get(2);
                int rand = (int)(Math.random() * range) + min;
                Values.addVar(var, String.valueOf(rand));
            }
        });
        out.put("math_round", new Function() {
            @Override
            public void invoke(List<String> args) {
                float num = Float.parseFloat(args.get(0));
                String var = args.get(1);
                Values.addVar(var, String.valueOf(num));
            }
        });
        out.put("math_sqr", new Function() {
            @Override
            public void invoke(List<String> args) {
                int a = Integer.parseInt(args.get(0));
                String var = args.get(1);
                Values.addVar(var, String.valueOf(a*a));
            }
        });
        return out;
    }
}

File: src/com/jepl/lang/libraries/math/math.kt
package com.jepl.lang.libraries.math

import com.jepl.lang.Values
import com.jepl.lang.libraries.Function
import com.jepl.lang.libraries.Library

class math : Library {
    override fun invoke(): HashMap<String, Function> {
        val out = java.util.HashMap<String, Function>()
        out["math_random"] = object : Function {
            override fun invoke(args: List<String>) {
                val range = args[1].toInt()
                val min = args[0].toInt()
                val `var` = args[2]
                val rand = (Math.random() * range).toInt() + min
                Values.addVar(`var`, rand.toString())
            }
        }
        out["math_round"] = object : Function {
            override fun invoke(args: List<String>) {
                val num = args[0].toFloat()
                val `var` = args[1]
                Values.addVar(`var`, num.toString())
            }
        }
        out["math_sqr"] = object : Function {
            override fun invoke(args: List<String>) {
                val a = args[0].toInt()
                val `var` = args[1]
                Values.addVar(`var`, (a * a).toString())
            }
        }
        return out
    }
}